src/newlines.cpp

```diff
@@ -397,6 +396,7 @@ chunk_t *newline_add_between(chunk_t *start, chunk_t *end)
    }
 
    /* Scan for a line break */
+   chunk_t *pc;
```

Comment:
The scopes could be limited a bit more if this definition is defined in the for loop and inside:
```
if (end->type == CT_BRACE_OPEN)
{
   pc = chunk_get_next(end);
```